src/hooks/useConfigure.ts

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useEffect } from 'react';
+import { Alert, useColorScheme } from 'react-native';
+import { SUBSCRIBER_EVENTS } from '@walletconnect/core';
+import { ExplorerCtrl } from '../controllers/ExplorerCtrl';
+import { OptionsCtrl } from '../controllers/OptionsCtrl';
+import { ConfigCtrl } from '../controllers/ConfigCtrl';
+import { ClientCtrl } from '../controllers/ClientCtrl';
+import { AccountCtrl } from '../controllers/AccountCtrl';
+import { WcConnectionCtrl } from '../controllers/WcConnectionCtrl';
+import type { ProviderMetadata } from '../types/coreTypes';
+import { createUniversalProvider } from '../utils/ProviderUtil';
+
+interface Props {
+  projectId: string;
+  providerMetadata: ProviderMetadata;
+  relayUrl?: string;
```

Comment:
NonBlocking: But as we get into theming, I assume theming will be a prop we will pass through.

Reply:
yes, it'll use the device theme as default but we'll let the user override it